backend/templates/emails/ticket_transfer.html

```diff
@@ -0,0 +1,27 @@
+<!-- TYPES:
+sender_first_name:
+    type: string
+receiver_first_name:
+    type: string
+receiver_username:
+    type: string
+event_name:
+    type: string
+type:
+    type: string
+-->
+
+{% extends 'emails/base.html' %}
+
+{% block content %}
+<h2>Thanks for using Penn Clubs!</h2>
```

Comment:
nit: do we have a thanks message in any of our other email templates? Haven't seen this before, just curious.

Reply:
It's in the ticket confirmation email which I copied, doesn't seem as appropriate for ticket transfers though...will remove